olad/www/new/js/app.js

```diff
@@ -310,9 +310,56 @@ ola.controller('keypadUniverseCtrl',
         } else if (check.regexGroups(fields)) {
           $scope.field = fields[0];
         }
+        $scope.focusInput = true;
+      };
+
+      $scope.keypress = function($event) {
+        var key = $event.key;
+
+        if ($event.altKey || $event.ctrlKey || $event.metaKey
+            || ($event.which == 0 && key != "Enter" && key != "Backspace")) {
+          // don't handle keyboard shortcuts and F1 - F12 keys
+          return;
+        }
+
+        $event.preventDefault();
+
+        switch (key) {
+          case '0':
+          case '1':
+          case '2':
+          case '3':
+          case '4':
+          case '5':
+          case '6':
+          case '7':
+          case '8':
+          case '9':
+            $scope.input(key);
+            break;
+          case '@':
+          case 'a':
+            $scope.input(' @ ');
+            break;
+          case '-':
```

Comment:
It was popular on Strand, and probably other consoles. The above would select channels 1, 2, 3, 4, 6, 7, 8, 9, 10 and 15.